client.js

```diff
@@ -162,14 +164,24 @@ function streamAuction (auction_user, auction_contents, server) {
     let channelID;
 
     if (server === "GREEN") {
-        channelID = "672512233435168784";
+        // channelID = "672512233435168784";
+        //for test server user...
+        channelID = "716521067388338207";
```

Comment:
Eugh, forgot to move channelID over to the discord settings file. Need to do that.